frontend/src/components/Year/Year.tsx

```diff
@@ -4,37 +4,80 @@ import { YearTop } from ".";
 import { SemesterBlock } from "../";
 import { DNDSchedule } from "../../models/types";
 import { SEMESTER_MIN_HEIGHT } from "../../constants";
+import KeyboardArrowDownIcon from "@material-ui/icons/KeyboardArrowDown";
+import KeyboardArrowUpIcon from "@material-ui/icons/KeyboardArrowUp";
+import { isYearInPast } from "../../utils";
 
-export interface IYearProps {
+interface Props {
   index: number;
   schedule: DNDSchedule;
 }
 
+interface State {
+  expanded: boolean;
+}
+
+const YearTopWrapper = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  align-items: center;
+  margin-left: -28px;
+`;
+
 const YearBody = styled.div`
   display: flex;
   flex-direction: row;
   min-height: ${SEMESTER_MIN_HEIGHT}px;
 `;
 
-export class Year extends React.Component<IYearProps> {
+export class Year extends React.Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+
+    const year = props.schedule.years[props.index];
+
+    this.state = {
+      expanded: !isYearInPast(year),
```

Comment:
It might be more accurate to use the academic year that the user enters (since if they're on their 3rd year, it's safe to assume you can collapse year 1 and 2)